app/views/playerControlsExtended.js

```diff
@@ -104,9 +104,16 @@ const ControlsExtended = Mn.ItemView.extend({
     changeVolWheel(e) {
         const wheelStep = 0.05;
         let volume = application.appState.get('currentVolume', volume);
-        if (e.originalEvent.wheelDelta > 0 ) {
-            volume += wheelStep;
+        let wheeldelta;
+        if (/firefox/i.test(window.navigator.userAgent)) {
```

Comment:
Sniffing userAgent isn't really reliable, as we may assume the user agent isn't lying on itself.

I suggest we find the delta by doing:

```js
const delta = e.originalEvent.wheelDelta || -e.originalEvent.deltaY
```

It's a smarter solution to support both mouseWheel interface and legacy DomScrollWheel 😉